ur_robot_driver/doc/resources/README.md

```diff
@@ -0,0 +1,30 @@
+# Resources about ROS 2 Universal Robot driver
+
+The files in this folder are available for use under [![License](https://i.creativecommons.org/l/by/4.0/88x31.png)](http://creativecommons.org/licenses/by/4.0/) license.
+
+
+## Diagrams and images
+
+- [ros2_control diagrams](ros2_control_ur_driver.drawio) - `.drawio` file, Author: [Denis Stogl](mailto:denis.stogl@picknik.ai) (PickNik Robotics)
```

Comment:
I tried download the .drawio file and loading it in https://app.diagrams.net/ and got this error

```
Error loading file
Not a diagram file (error on line 15 at column 76: Specification mandates value for attribute crossorigin)
```

Am I doing something wrong? I am not familiar with drawio files.

Reply:
I just tested it and it works fine.

How did you download the file? I used `wget` and a link to the raw file. Than drag-and-droped it to the diagrams.net